#158792524 Wireframe of the product

Opio-Emmanuel-Omona opened the pull request:
#### What does this PR do?
- [x] This PR creates a wireframe for the application.

#### Description of Task to be completed?
- [x] Create a wireframe directory
- [x] Create a wireframe on [marvel app](https://marvelapp.com/)

#### How should this be manually tested?
* [Click this link to the wireframes and navigate through the app.](https://marvelapp.com/b6098ja/screen/54563142)

#### What are the relevant pivotal tracker stories?
[ #158792524]( https://www.pivotaltracker.com/story/show/158792524)

#### Screenshots 
![wireframe](https://user-images.githubusercontent.com/26769886/54591883-39386080-4a3c-11e9-872f-64a72e38909e.gif)

PromasterGuru commented:
Good Job on the design. 
Basically, the app will have two activities i.e. the Main
Activity displaying the list of Github users as shown on your mockup and the DetailActivity displaying details of single user selected from the MainActivity. The share button should, therefore, be on the DetailActivity. In addition, add the images on your README.md file instead of the links. This will save you from getting a lot of feedbacks on your LMS outputs.

Opio-Emmanuel-Omona commented:
https://github.com/Opio-Emmanuel-Omona/code-labs/pull/1#issuecomment-474257817

@PromasterGuru, I have changed the wireframe to contain 2 activities and also added images to the ReadME file.
